plugins/modules/ipaconfig.py

```diff
@@ -472,6 +472,9 @@ def main():
     allow_empty_string = ["pac_type", "user_auth_type", "configstring"]
     reverse_field_map = {v: k for k, v in field_map.items()}
 
+    if ansible_module.params_get("enable_sid") is False:
```

Comment:
Shouldn't this only fail if SID is enabled? If SID is not enabled if should be ok to have this set to False explicitly.